App crash

Reproduce:
1.open App 
2. Click on folder
3.menu Sync this folder 

Except:
The folder is syncing

Server
Nextcloud 18.0.4




************ CAUSE OF ERROR ************

java.lang.RuntimeException: Unable to start service com.owncloud.android.services.OperationsService@7034c8b with Intent { act=SYNC_FOLDER cmp=com.nextcloud.client/com.owncloud.android.services.OperationsService (has extras) }: java.lang.ClassCastException: com.nextcloud.client.account.RegisteredUser cannot be cast to android.accounts.Account
	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:4235)
	at android.app.ActivityThread.access$2200(ActivityThread.java:231)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1997)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7695)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:516)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
Caused by: java.lang.ClassCastException: com.nextcloud.client.account.RegisteredUser cannot be cast to android.accounts.Account
	at com.owncloud.android.services.OperationsService.onStartCommand(OperationsService.java:196)
	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:4217)
	... 8 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30120051
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: OnePlus
Device: OnePlus7TPro
Model: HD1913
Id: QKQ1.190716.003
Product: OnePlus7TPro

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 2003271900

--- a/src/main/java/com/owncloud/android/operations/SynchronizeFolderOperation.java
+++ b/src/main/java/com/owncloud/android/operations/SynchronizeFolderOperation.java
@@ -491,7 +491,7 @@ public class SynchronizeFolderOperation extends SyncOperation {
     private void startSyncFolderOperation(String path){
         Intent intent = new Intent(mContext, OperationsService.class);
         intent.setAction(OperationsService.ACTION_SYNC_FOLDER);
-        intent.putExtra(OperationsService.EXTRA_ACCOUNT, user);
+        intent.putExtra(OperationsService.EXTRA_ACCOUNT, user.toPlatformAccount());
         intent.putExtra(OperationsService.EXTRA_REMOTE_PATH, path);
         mContext.startService(intent);
     }

